src/components/organisms/TCRecommendedUserToFollowList/TCRecommendedUserToFollowList.tsx

```diff
@@ -0,0 +1,28 @@
+import { FunctionComponent } from 'react'
+import { RecommendedUserToFollow } from '../../../model/RecommendedUserToFollow'
+import { TCRecommendedUserToFollow } from '../../molecules/TCRecommendedUserToFollow/TCRecommendedUserToFollow'
+import './TCRecommendedUsersToFollowList.scss'
+
+interface TCRecommendedUsersToFollowlistProps {
+  users: RecommendedUserToFollow[]
+}
+
+export const TCRecommendedUsersToFollowlist: FunctionComponent<
+  TCRecommendedUsersToFollowlistProps
+> = ({ users }) => {
+  return (
+    <div className='recommendedUsersToFollow'>
+      <div className='header'>
+        <p className='header__title'> Who to follow </p>
+      </div>
+
+      {users.map((user) => (
```

Comment:
I would also wrap this in div with style Should be tc-recommended-users-to-follow-list__users